style.css

```diff
@@ -1,825 +1,810 @@
+    height: 92px;
+    position: relative;
   }
 
-  .first__sidebar-social-inst,
-  .first__sidebar-social-vk,
-  .first__sidebar-social-fb {
-    width: 35px;
-    height: 35px;
+  .header__inner-menu-btn::after {
+    content: '';
+    position: absolute;
+    width: 34.7px;
+    height: 3.48px;
+    background-color: #000;
+    top: 39.54px;
+    left: 28.25px;
+    box-shadow: 0 9.44px;
   }
 
-  .first__sidebar-social {
-    width: 123px;
-    padding: 652px 45px 0;
+  .head__desctop {
+    display: block;
   }
-}
 
-@media (max-width: 1000px) {
   .first {
-    max-width: 1000px;
-    width: 100%;
+    min-width: 957px;
     display: grid;
-    grid-template-columns: 35.2% 64.8%;
-    grid-template-rows: 541px;
-  }
-
-  .header__logo {
-    font-size: 17px;
-    letter-spacing: 2px;
-    margin: 4px 31px;
-  }
-
-  .header__menu {
-    height: 54px;
-    width: 54px;
+    grid-template-columns: 32% 68%;
+    grid-template-rows: 900px;
   }
 
-  .header__menu-item {
-    height: 3px;
-    width: 22.7px;
-    margin-bottom: 4px;
+  .first__sidebar {
+    grid-column: 1;
+    background: black;
   }
 
-  .header__language {
-    left: 29.5%;
-    top: 24px;
-    font-size: 12px;
+  .first__sidebar-social {
+    display: flex;
+    justify-content: space-between;
+    width: 155px;
+    padding: 816px 51px 0;
   }
 
-  .header__vector {
-    left: 32%;
-    top: 31px;
-    width: 9px;
-    height: 6px;
+  .first__sidebar-social-vk,
+  .first__sidebar-social-fb {
+    margin-left: 12px;
   }
 
-  .img__img {
-    width: 329px;
-    height: 338px;
+  .first__collection {
+	  background: #fff;
   }
 
   .step {
     position: absolute;
     width: 100%;
     display: grid;
-    grid-template-columns: 329px 329px;
-    grid-template-rows: 338px;
+    grid-template-columns: 439px 439px;
+    grid-template-rows: 454px;
     justify-content: center;
-    top: 101px;
+    top: 168px;
     z-index: 2;
-    grid-column-gap: 31px;
+    grid-column-gap: 54px;
   }
 
-  .text {
-    width: 329px;
-    height: 338px;
+  .img__img {
+    height: 450px;
   }
 
-  .text__title {
-    font-size: 49px;
-    line-height: 50px;
-    letter-spacing: 1px;
-    width: 329px;
+  .text {
+    height: 450px;
   }
 
   .text__container {
-    padding: 27px 0;
+    padding: 80px 0;
   }
 
-  .text__subtitle {
-    font-size: 9px;
-    line-height: 13px;
-    letter-spacing: 1px;
-    margin-bottom: 24px;
+  .text__title {
+    font-size: 60px;
+    line-height: 68px;
+    letter-spacing: 4px;
+    width: 439px;
   }
 
-  .collection__next-text {
-    font-size: 8px;
-    padding: 7px 22px;
-    margin-right: 31px;
-    letter-spacing: 0;
+  .text__subtitle {
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 1.1px;
+    margin-bottom: 40px;
   }
 
-  .collection__other-text {
-    font-size: 8px;
-    letter-spacing: 0;
-    margin-right: 3px;
-  }
+.collection__next-text {
+  text-decoration: none;
+  color: black;
+  font-style: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)